pkg/daemon/ceph/client/pool.go

```diff
@@ -264,17 +264,12 @@ func CreateReplicatedPoolForApp(context *clusterd.Context, namespace, poolName s
 		return err
 	}
 
-	args := []string{"osd", "pool", "create", poolName, pgCount, "replicated", poolName}
+	args := []string{"osd", "pool", "create", poolName, pgCount, "replicated", poolName, "--size", strconv.FormatUint(uint64(pool.Replicated.Size), 10)}
 	output, err := NewCephCommand(context, namespace, args).Run()
 	if err != nil {
 		return errors.Wrapf(err, "failed to create replicated pool %s. %s", poolName, string(output))
 	}
 
-	// the pool is type replicated, set the size for the pool now that it's been created
-	if err = SetPoolReplicatedSizeProperty(context, namespace, poolName, strconv.FormatUint(uint64(pool.Replicated.Size), 10)); err != nil {
```

Comment:
We still need thin. In case the replicated.size is changed for a pool, this code will run again to set the new size. Since it was removed, the pool resize integration test is failing. 

So if we do both actions to pass the `--size` param during pool creation and also call this function to adjust the pool size, we will get the min size properly computed at pool creation, but still have the ability to change the pool size.

Reply:
Make sense.

I restored `SetPoolReplicatedSizeProperty` block.